tests/src/csv/LineObject.cpp

```diff
@@ -0,0 +1,196 @@
+#include <sstream>
+#include <string_view>
+
+#include <openvic-dataloader/csv/LineObject.hpp>
+
+#include <range/v3/range/primitives.hpp>
+#include <snitch/snitch.hpp>
+
+using namespace ovdl;
+using namespace csv;
+using namespace std::string_view_literals;
+
+TEST_CASE("LineObject", "[line-object]") {
```

Comment:
Nice tests. Once I adjusted to the format, I quickly understood what each test does.

Reply:
Yeah snitch is a modern C++20 testing suite based on some of the interface to Catch2.